warpcast-langchain/warpcast_api_wrapper.py

```diff
@@ -0,0 +1,37 @@
+import requests
+
+
+class WarpcastApiWrapper:
+    def __init__(self, api_key: str):
+        self.api_key = api_key
```

Comment:
How is the API Key constructed? [Farcaster Auth](https://docs.farcaster.xyz/reference/warpcast/api#authentication) requires a self signed access token. If this has changed, can we please add instructions to the README for generating the correct API Key.